src/main/java/frc/team2412/robot/subsystems/LimelightSubsystem.java

```diff
@@ -0,0 +1,133 @@
+package frc.team2412.robot.subsystems;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LimelightSubsystem extends SubsystemBase {
+
+	// POTENTIAL CONSTANTS
+	public static final double GOAL_DISTANCE_FROM_NOTE = 20.0;
```

Comment:
Ideally this constant would just be the default value, and the actual value used in `isWithinDistance()` is a Shuffleboard-configurable value, so it's fast and easy to change it and retest without having to redeploy robot code.